client/dist/styles.css

```diff
@@ -1,7 +1,138 @@
+/* RESET */
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+
+/* kevin's CSS */
 h1 {
   color: red;
 }
 
 p {
   color:black;
-}
\ No newline at end of file
+}
+
+.review-summary{
+  font-weight: bold;
+}
+
+.review-list {
+  height: 1000px;
+  overflow-y: scroll;
+
+}
+
+/* review tile body */
+.review-tile {
+  margin: 1em;
+  border: .25em solid red;
+  /* overflow-y: scroll; */
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
```

Comment:
- looks like you have a strategy for drawing/filling stars... that's a lot of code. I have not done any css yet, but I think I will look for some alternative star ideas just in case there's a shorter way to go about it.